Sources/ContainerBuilder/ContainerBuilder.swift

```diff
@@ -237,7 +237,7 @@ private func sha256(data: Data) -> String {
 /// - Parameter destinationURL: The URL to save the tarball to.
 /// - Throws: An error if the tarball cannot be created.
 private func createTarball(from sourceDir: URL, to destinationURL: URL) async throws {
-    _ = try await Subprocess.run(
+    _ = try await Subprocess.run(Z
```

Comment:
am I reading this line right? there's a Z added..?

Reply:
Huh, no that shouldn't have been there. I sometimes but these marks in there while committing as I'm still writing in the editor...